test/junit/src/test/scala/strawman/collection/SeqViewTest.scala

```diff
@@ -11,6 +11,7 @@ class SeqViewTest {
   @Test
   def test_SI8691(): Unit = {
     // Really just testing to make sure ++: doesn't throw an exception
-    assert( (Seq(1,2) ++: Seq(3,4).view).to(Seq) == Seq(1,2,3,4) )
+    assert( (Seq(1,2) :++ Seq(3,4).view).to(Seq) == Seq(1,2,3,4) )
+    assert( (Seq(1,2).view ++: Seq(3,4)).to(Seq) == Seq(1,2,3,4) )
```

Comment:
`SeqView` doesn’t have `++:` anymore?

Reply:
There is no `SeqView`. The `view` method only ever returns `View` (in `Iterable`),  `IndexedView` (in `IndexedSeqOps`), `ArrayView` (in `ArrayOps`), `StringView` (in `StringOps`),  `ArrayBufferView` (in `ArrayBuffer`) and of course `View`(in `View`).

Before this PR `++:` was available on `Iterable` and thus also on `View`, to make it available again we would need a `SeqView` that extends `SeqOps` and which is returned by the `view` method in `Seq`.

Do you want me to add `SeqView` in this PR (as it is a consequence of moving `prepend` & friends from `Iterable` to `Seq`)?

By the way, I also noticed that `IndexedView` does not extend `IndexedSeqOps` but instead `SeqOps` directly (it does not make much difference however since `IndexedSeqOps` contains no new operations, only overrides for performance, but it does look a bit unregular). Is this intended or should it be changed?

Reply:
Oh, I thought we had `SeqView` :( We should probably add it, so that views of collections that inherit from `Seq` have the same operations.

Reply:
You are right, `IndexedView` should extend `IndexedSeqOps`.